src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -33,230 +14,41 @@
-                       + "</td><td> <br/> &nbsp;</td></tr>");
-        CommonUtils.appendTextToFile(logPath, builder.toString());
-      }
-    }
-  }
-
-  public static List<Boolean> getVerificationStack() {
-    return logsResults;
-  }
-
-  public static void start(Method testMethod) {
-    StringBuilder builder = new StringBuilder();
-    String testName = testMethod.getName();
-    String className = testMethod.getDeclaringClass().getCanonicalName();
-
-    builder.append("<table>" + "<h1>Class: <em>" + className + "." + testName + " </em></h1>");
-    if (testMethod.isAnnotationPresent(RelatedIssue.class)) {
-      String issueID = testMethod.getAnnotation(RelatedIssue.class).issueID();
-      String jiraUrl = jiraPath + issueID;
-      builder.append("<tr class=\"step\"><td>Known failure</td><td><h1><em>" + testName + " - "
-                     + "<a href=\"" + jiraUrl + "\">" + issueID + "</a> "
-                     + testMethod.getAnnotation(RelatedIssue.class).comment()
-                     + "</em></h1></td><td> <br/> &nbsp;</td></tr>");
-    } else {
-      builder.append("<tr class=\"step\"><td>&nbsp</td><td><h1><em>" + testName
-                     + "</em></h1></td><td> <br/> &nbsp;</td></tr>");
-    }
-    CommonUtils.appendTextToFile(logPath, builder.toString());
-    testStarted = true;
-  }
-
-  public static void stopLogging() {
-    StringBuilder builder = new StringBuilder();
-    builder.append("<tr class=\"step\">" + "<td>&nbsp</td><td>STOP LOGGING METHOD  "
-                   + "<div style=\"text-align:center\">" + "<a href=\"#toc\" style=\"color:blue\">"
-                   + "<b>BACK TO MENU</b></a></div> </td><td> <br/> &nbsp;</td></tr>" + "</table>");
-    CommonUtils.appendTextToFile(logPath, builder.toString());
-    testStarted = false;
-  }
 
   @Override
   public void beforeNavigateTo(String url, WebDriver driver) {
-    StringBuilder builder = new StringBuilder();
-    builder.append(
-        "<tr class=\"success\"><td>Navigate to</td><td>" + "<a href='" + url + "'>" + url
-        + "</a></td><td> <br/> &nbsp;</td></tr>");
-    CommonUtils.appendTextToFile(logPath, builder.toString());
-    logJSError(driver);
+    LOG.info("Navigate to", url);
+    LOG.logJSError();
   }
 
   @Override
   public void afterNavigateTo(String url, WebDriver driver) {
-    StringBuilder builder = new StringBuilder();
+    try {
+      Thread.sleep(1000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
+    }
     if (!AlertHandler.isAlertPresent(driver)) {
       if (url.equals(driver.getCurrentUrl())) {
-        builder.append("<tr class=\"success\"><td>Url after navigation</td><td>"
-                       + "<a href='" + driver.getCurrentUrl() + "'>" + driver.getCurrentUrl()
-                       + "</a></td><td> <br/> &nbsp;</td></tr>");
-        CommonUtils.appendTextToFile(logPath, builder.toString());
+        LOG.info("Url after navigation", driver.getCurrentUrl());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Url after navigation" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)